e2e-test/src/it/scala/io/scalac/mesmer/e2e/ExampleAkkaTest.scala

```diff
@@ -0,0 +1,26 @@
+package io.scalac.mesmer.e2e
+
+import org.scalatest.wordspec.AnyWordSpec
+
+class ExampleAkkaTest extends AnyWordSpec with E2ETest with PrometheusMetrics {
+
+  // Akka example does not produce following metrics:
+  // mesmer_akka_actor_failed_messages
+  // mesmer_akka_actor_unhandled_messages
+  // mesmer_akka_actor_dropped_messages
+  private val akkaMetrics = (Seq(
+    "mesmer_akka_actor_mailbox_size",
+    "mesmer_akka_actor_stashed_messages",
+    "mesmer_akka_actor_sent_messages",
+    "mesmer_akka_actor_actors_created",
+    "mesmer_akka_actor_actors_terminated"
```

Comment:
I think you can add the persistence and cluster metrics here as well

Reply:
I cannot see neither cluster nor persistence metrics produced by the Akka example. Is something missing in its setup?

Reply:
Hmm... it might be the case that you need to send a rest request to generate persistence metrics. Maybe we can rebuild the example so that the rest call is not needed? When it comes to cluster metrics - it's odd that they're not there.